docs/dev/api/insights.md

```diff
@@ -1024,22 +1105,26 @@ Return daily statistics about test results and concurrency usage for the specifi
   </tbody>
   <tbody>
     <tr>
-     <td><code>since</code></td>
-       <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>Return only jobs ran from the provided Unix timestamp on.</p></td>
+     <td><code>group_id</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>Return results only for the specified <code>group_id</code>.</p></td>
     </tr>
   </tbody>
   <tbody>
     <tr>
-     <td><code>until</code></td>
-       <td><p><small>| QUERY | OPTIONAL | STRING|</small></p><p>Return only jobs ran until the provided Unix timestamp.</p></td>
+     <td><code>team_id</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>Return results only for the specified <code>team_id</code>.</p></td>
     </tr>
   </tbody>
   <tbody>
     <tr>
-     <td><code>level</code></td>
-     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>
-     Specifies the ownership level. Supported values are: <code>user</code>, <code>organization</code>
-     </p></td>
+     <td><code>must_have</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>[Add description]. Available value is: <code>exception</code>.</p></td>
```

Comment:
@LoryApiFortress Looks like the description is missing for `must_have`.

Reply:
@lysannep I know...I'm waiting for @paweldudzinski to provide the description.